Superlista/resources/productList_en.json

```diff
@@ -0,0 +1,1533 @@
+  },
+  {
+    "id": 40,
+    "name": "Peanut",
+    "category": "Bomboniere"
+  },
+  {
+    "id": 41,
+    "name": "Straw potato",
+    "category": "Bomboniere"
+  },
+  {
+    "id": 42,
+    "name": "Marshmallow",
+    "category": "Bomboniere"
+  },
+  {
+    "id": 43,
+    "name": "Beef",
+    "category": "Meats"
+  },
+  {
+    "id": 44,
+    "name": "Chicken",
+    "category": "Meats"
+  },
+  {
+    "id": 45,
+    "name": "Ground beef",
+    "category": "Meats"
+  },
+  {
+    "id": 46,
+    "name": "Sausage",
+    "category": "Meats"
+  },
+  {
+    "id": 47,
+    "name": "Bacon",
+    "category": "Meats"
+  },
+  {
+    "id": 48,
+    "name": "Fish",
+    "category": "Meats"
+  },
+  {
+    "id": 49,
+    "name": "Shrimp",
+    "category": "Meats"
+  },
+  {
+    "id": 50,
+    "name": "Pork",
+    "category": "Meats"
+  },
+  {
+    "id": 51,
+    "name": "Sheep",
+    "category": "Meats"
+  },
+  {
+    "id": 53,
+    "name": "Raisins",
+    "category": "Jams and sweets"
+  },
+  {
+    "id": 54,
+    "name": "Dried fruits",
+    "category": "Jams and sweets"
+  },
+  {
+    "id": 55,
+    "name": "Syrup",
+    "category": "Jams and sweets"
+  },
+  {
+    "id": 56,
+    "name": "Hazelnut cream",
+    "category": "Jams and sweets"
+  },
+  {
+    "id": 57,
+    "name": "Sweet milk",
+    "category": "Jams and sweets"
+  },
+  {
+    "id": 58,
+    "name": "Jam",
+    "category": "Jams and sweets"
+  },
+  {
+    "id": 60,
+    "name": "Chicken breaded",
+    "category": "Frozen"
+  },
+  {
+    "id": 61,
+    "name": "Pizza",
+    "category": "Frozen"
+  },
+  {
+    "id": 62,
+    "name": "Lasagna",
+    "category": "Frozen"
+  },
+  {
+    "id": 63,
+    "name": "Frozen food",
+    "category": "Frozen"
+  },
+  {
+    "id": 64,
+    "name": "Ice cream",
+    "category": "Frozen"
+  },
+  {
+    "id": 65,
+    "name": "Acai",
+    "category": "Frozen"
+  },
+  {
+    "id": 66,
+    "name": "Frozen vegetables",
+    "category": "Frozen"
+  },
+  {
+    "id": 67,
+    "name": "Meatballs",
+    "category": "Frozen"
+  },
+  {
+    "id": 68,
+    "name": "Hamburger",
+    "category": "Frozen"
+  },
+  {
+    "id": 69,
+    "name": "Cheese bread",
+    "category": "Frozen"
+  },
+  {
+    "id": 70,
+    "name": "French fries",
+    "category": "Frozen"
+  },
+  {
+    "id": 71,
+    "name": "Oil",
+    "category": "Canned, pickled and oils"
+  },
+  {
+    "id": 72,
+    "name": "Funghi",
+    "category": "Canned, pickled and oils"
+  },
+  {
+    "id": 73,
+    "name": "Sardine",
+    "category": "Canned, pickled and oils"
+  },
+  {
+    "id": 74,
+    "name": "Pickled cucumbers",
+    "category": "Canned, pickled and oils"
+  },
+  {
+    "id": 75,
+    "name": "Tuna",
+    "category": "Canned, pickled and oils"
+  },
+  {
+    "id": 76,
+    "name": "Quail eggs",
+    "category": "Canned, pickled and oils"
+  },
+  {
+    "id": 77,
+    "name": "Pickled onions",
+    "category": "Canned, pickled and oils"
+  },
+  {
```

Comment:
{
     "id": 78,
     "name": "Canned Pate",
     "category": "Canned, pickled and oils"
   },